.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npx lint-staged
```

Comment:
`npx` involves runnnig code that's not verified against hashes in yarn.lock. Doing this in ci is a risk we do not usually take, I think.

Reply:
I'll replace it with a lint check. I wasn't aware of this - thanks for letting me know.

Reply:
In this case, I suppose `lint-staged` _is_ in `yarn.lock`. But one typo and you're running `lint-stoged` which does who-knows-what.